.travis.yml

```diff
@@ -4,11 +4,50 @@ env:
+  - secure: epiUwtY61ipALmWhfo+0JkRTaItcu7ua8clX1GfoZYMlFl0RrKmQRfeDxXRBzugzGgdkhzFwj3eWJ2y0HKg8th1m81NvRHPLoNs3sz4r5dQabhPPQhc5uihmY2TfI75pixetnsn/1BMSrz4ld05bAemPS3yOtbdNQMSA4AjN7jfAHBopbSyNXAvzLS64ppmwMRFnQmstE1dp3q7xCyybVMkzV9styGspAoEP/DXYb+wIBMn/nP5SSKhZUfkGKBRclTXim9AjDSAtSISzgwYyqT/puNmBUe8A0byk/i4reOUdl1av8Oo7BgHkOwOyWPQAwfKAtLxj4Lo2AUqxukb06L7LY9xaMRw1MosoHcylsFyqqvFVe6YjCb4c1i6h8yrkzgcsow1IVc7d1iCxcesWaaSAMm+3FcIAvXlJ7b7lE6REJYv047HdKchyi92A4sav6Wj92VKNB9iakRXYsfWHJM6NO/5jAgqtpl6VPB3Iwo6fUB3qf60PsDaxAwCicAQzYw40QOsGLvjQ0wPVMYUga8EFiZO/vZP9StGExgq0+vZ8LK1BirQ+wGQdm7l1HUywlbT36YIBFol+7fE239NFxNLNEaQyXJjCg3Pe0HbHA9/rOLCCA5PBdpHSc5Dcj5S4sSgvgIar7/Vv42plcUJI3n28U/Ie3Wujc2flPHsAL0A=
+ 
+  - DOCKER_USERNAME=olutiab
+  - DOCKER_PASSWORD=
+ 
+  - secure: M0c6J193KYGEC2KdzSGuzs/05APm9SDzQOQ5Vq2cfxtb5pPOHoYjB1llA0056QHBcBjMJ/Md6o6mrQm4kLlXStczfphYb7KSM0em4KeQUhF0BE2TuS2EFEATy3rpIslfa+TeWxAGhIJegvMNGT99ec+4iBGcPRQYLmLu3AO2duC7YB0wU3YRYq5VuHPvL2JYi0qqeMrr29BlX0NHN18UaMGN2gPHQUDGH3RNKs4p3Bnb70tje4PRyZVhH+4wxakYdx47KBNziIdthfZ/lpYrWtDqfcWn+w3lUOjeadxMDYcZzPT7jycs5k6eLn+Js9QO1AdWPMvt7zdy7vmsUtqIhGIID0Sj6IW/fWk3B0P9z2IajbGHZbNWM8/NAw+mK30cWRe6LLR3m8dZ/YnHD2eZjG6N2E2g5EnIWpTis9GeEVGm9vHW/s6vRYO2VriD6Rc9hIuZrOC6piWN+MemXM8s9vhuScmYNh1XnDk76vBXhBeelv6ov8HRl1654LS3ydVem7UbHvX4299DsYdUq9H55gbPCIfNV9lUes2+CG+7mW7TMJGaIN1dzk3FEgiaTmEzHO36I1mNvaXjep0/UFS9NljZm/EfY0G0sUY968OCmhlEDTNhMhkshwAxjpv8SdsLs6+2Ywy8mHnqYH/fIsrIFG0AaiOaUPiXy9xJUxtkZTQ=
+ 
+  - HEROKU_API_KEY=
+  - secure: hvvkAPY2iPtpqbIQ0OkutUEVyzSQ24vUiBrThYvndv8WazbCAt98nVWFYwI7pF8qq/8/rdni1bEuT+4C1tCnJ2W9cv3BNkzbjE5L41VNhTgrUCFED+KAsiFBTWML+1GtgYz7hlpJiHNGkQzCl5+Mue2z6mXeydfuV0Wsilfuh7VsMdIDWjoZrLaUNoNrF81RTIAVmdJKlnkbhdGNe3+AGLosxF6MNtbeLvQ/TcxQakmlaS/V4/c39wGNKcNxqxC1vbDVXKJKCNxzbBlTb7O5jBMoGxiN3e2GjMBzVF+TZl4YJX4EPa7EX7qkUeVcxxxYOTZ4bHTORN02oB5vAwBRL+Oy8/9v1IzoC1r3m4W62MPuC3ICI03X1+/aJB6xIyOc8Yf0jH+9Kk5uyN6GfS2KJWPJrbVy2PvA7RS0LidG+2lw7rUstL7GKq7gbXucJXzX7LaEV2H8LitOo4mI1V/yOz3gS1aWsTtmdcBm+pcEQK3XJhqReoweVn73zd1XKJNui1vxqd+paQxwd3zHAtBjQSs0dw8ejF0Uz84uY0Ilptw0ntFIdJK0EnncxN04qkuGb06ca1y1P7gTajX+3MQ2JhD32tScnPvblWpODoVfHjabcCQYguqdiR/NtDl/FDquTNPDYUGP9Z3aRaKK2zBV2PTBm/W7j2OJ+53sAbv5ZYg=
+
 script:
+- echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+
+- docker pull olutiab/my-heroku-build
 - docker build --target test --tag my-test-image .
+
 - docker run my-test-image tests/test_view_model.py
 - docker run my-test-image tests/test_integration.py
 - docker run -e TRELLO_API_KEY -e TRELLO_API_TOKEN my-test-image tests/test_selenium.py
+
+- docker build --target production --tag olutiab/my-heroku-build .
+
+- docker tag olutiab/my-heroku-build registry.heroku.com/olutiab-todo-app/web
+
+- docker push registry.heroku.com/olutiab-todo-app/web
```

Comment:
You'll need to run a `docker login` for the heroku registry for this to work e.g.:
`docker login --username=$HEROKU_LOGIN --password=$HEROKU_API_KEY registry.heroku.com`
However I'd be tempted to move this line to `deploy.sh` and along with the `docker login` code provided here.